OpenRA.Mods.Common/Activities/Air/ReturnToBase.cs

```diff
@@ -135,10 +146,40 @@ public override Activity Tick(Actor self)
 				var nearestResupplier = ChooseResupplier(self, false);
```

Comment:
Line 144: Doesn't `ChooseResupplier` already guarantee that `dest.IsDead` is false?

Reply:
Yeah, updated.